features/DVO_Recommendations/Smart_Proxy_REST_API.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Behaviour specification for new REST API endpoints that will be implemented in Smart Proxy:
+
+        GET /namespaces/dvo
+        Return the list of all DVO namespaces (i.e. array of objects) to which this
+        particular account has access.  Each object contains the namespace ID, the
+        namespace display name if available, the cluster ID under which this namespace
+        is created, and the number of affecting recommendations for this namespace as
+        well.
+
+        GET /cluster/{cluster_name}/namespaces/dvo
+        GET /namespaces/dvo/cluster/{cluster_name}
```

Comment:
why are there 2 endpoints here?

Reply:
well both seems to make sense, and the handler will be the same

Reply:
usually if some "resources" are orthogonal, it's hard to set precedence rules